src/actions/guides/database/validating-saving.cr

```diff
@@ -436,6 +437,60 @@ class Guides::Database::ValidatingSaving < GuideAction
     end
     ```
 
+    #{permalink(ANCHOR_CHANGE_TRACKING)}
+    ## Tracking changed attributes
+
+    Sometimes you need to run code only when certain attributes have changed
+    (sometimes called "dirty tracking"). Avram Attributes have a `changed?`
+    and `original_value` method that makes it easy to see if an attribute has
+    changed.
+
+    The following change tracking methods are available:
+
+    * `changed?` - returns `true` if the attribute value has changed.
+    * `changed?(from: value)` - returns `true` if the attribute has changed
+      from the passed in value to anything else.
+    * `changed?(to: value)` - returns `true` if the attribute value has changed
+      to the passed in value.
+    * `original_value` - returns the original value before it was changed. If the
+      attribute is unchanged `value` and `original_value` will be the same.
+
+    > You can also combine `from` and `to` together: `name.changed?(from: nil, to: "Joe")`
+
+    Here is an example using `changed?` and `original_value` in an operation:
+
+    ```crystal
+    class SaveUser < User::SaveOperation
+      permit_columns name, email, admin
+
+      before_save do
+        if admin.changed?(to: true)
+          validate_company_email
+        end
+      end
+
+      def validate_company_email
+        if !email.ends_with?("@my-company.com")
+          email.add_error("must be from @my-company.com to be an admin")
+        end
+      end
+
+      after_save :log_changes
```

Comment:
```suggestion
      after_save log_changes
```

You might be able to do a symbol here, but other docs don't use it that way, so for consistency